core/src/commonMain/kotlin/com/revenuecat/purchases/kmp/PurchasesAreCompletedBy.kt

```diff
@@ -3,21 +3,52 @@ package com.revenuecat.purchases.kmp
 /**
  * Modes for completing the purchase process.
  */
-public expect enum class PurchasesAreCompletedBy {
+public sealed class PurchasesAreCompletedBy {
```

Comment:
Great idea! No reason to have it as a class if it can be an interface. Updated in https://github.com/RevenueCat/purchases-kmp/pull/139/commits/b0cac80e7c4133a2ce8952e215a0a3fbe222d80c